docs/references.md

```diff
@@ -38,3 +38,13 @@
 
 - [Ralf Gommers - Python packaging in 2021 - pain points and bright spots (2021)](https://labs.quansight.org/blog/2021/01/python-packaging-brainstorm/)
 
+
+## General packaging core concepts
+
+- [Python Packaging Authority - Python Packaging User Guide (2010)](https://packaging.python.org/)
```

Comment:
I agree with the comment about not using a date here, can you remove it? Let's keep maintained resources like these above the year-dated ones.

Reply:
And I think the scikit-HEP and py-pkgs.org ones are also maintained.